src/main/java/algorithms/minimumSpanningTree/prim/README.md

```diff
@@ -0,0 +1,46 @@
+# Prim's Algorithm
+
+## Background
+
+Prim's Algorithm is a greedy algorithm that finds the minimum spanning tree of a graph by starting from an
+arbitrary node (vertex) and adding the edge, with the minimum weight that connects the current tree to an unexplored
+node, and the unexplored node to the current tree, until all nodes are included in the tree.
+
+### Implementation Details
+
+A priority queue (binary heap) is utilised to keep track of the minimum weight edge that connects the current tree to an
```

Comment:
nice, but could you break it down into steps? so the chunk of text doesnt look too intimidating